server/server.js

```diff
@@ -40,6 +40,7 @@ app.use(bodyParser.json());
 app.use(express.static(path.join(__dirname, '../client/build')));
 
 const allowRoles = (...roles) => (token) => {
+  roles.push('superuser');
```

Comment:
Nitpick: Haven't checked, but the linter may complain about overwriting a param. We could use `if (![...roles, 'superuser'].some((role) => token.hasRole(role))) return false;` if this is the case.

Reply:
I could write this in a more clear way too, which would be fair:
The idea being that superusers can do everything - so check if the token hasRole superuser and return true if so, if not then continue on to checking the params